x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/cleaner/local/LocalIndicesCleanerTests.java

```diff
@@ -59,31 +55,4 @@ protected void assertIndicesCount(int count) throws Exception {
             assertThat(getSettingsResponse.getIndexToSettings().size(), equalTo(count));
         });
     }
-
-    public void testHandlesWatcherHistory() throws Exception {
```

Comment:
I think this entire test class can be removed? Since the only test method is now removed.

Reply:
There are test in `AbstractIndicesCleanerTestCase` that are still running and testing other parts of local cleaner, so I think this should stay